### PR TITLE
Updates account deletion copy & relevant FAQ section

### PR DESCRIPTION
**Pull request by rediris**
See https://deploy-preview-673--thegymcms.netlify.app/faq/#account-deletion

### CODE REVIEW

File: partials/account-deletion.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ---
 <div id="deletion-helper" class="hide" style="display: none;">
   <p>To delete your data, please fill out <a href="https://privacyportal.onetrust.com/webform/05f24bb5-a0c4-43d4-966e-05606d5be9ca/93b6b82e-baf8-4adc-8998-0df20831be6d" rel="noopener" target="_blank">our data deletion request form</a>.</p>
-  <p>Be advised that deleting your data will remove any certificates/badges that you have earned on the Gymnasium. Please back up (or download) your certificates/badges immediately if you plan on keeping them.</p>
+  <p>Heads up! Deleting your account data will remove any certificates/badges that you’ve earned on Aquent Gymnasium. We advise that you save them now. See our <a href="/faq#account-deletion">FAQ page</a> for more information.</p>
```
Comment (jgagne) [inline partials/account-deletion.html:7]: Consider hyphenation of, Heads-up!

File: static-pages/faq/index.html [hunk 2]
```diff
@@ -85,20 +85,21 @@ <h2>How Do I Change The Password On My Account?</h2>
     <p>Easily! Go to your student profile on the left-hand side of the <a href="/dashboard">Dashboard</a>. Click “Reset Password” and we’ll email you a link to reset your password.</p>
   </li>
 
-  <li>
+  <li id="account-deletion">
     <h2>How Can I Delete My Account?</h2>
-    <p>We understand, but we do hate to see you go. Send an email to <b><a href="{{ 'mailto:help@thegymnasium.com?subject=Get+In+Touch' | encode_email }}">{{ 'help@thegymnasium.com' | encode_email }}</a></b> and we’ll take care of everything for you.</p>
+    <p>We understand, but we do hate to see you go. Please fill out <a href="https://privacyportal.onetrust.com/webform/05f24bb5-a0c4-43d4-966e-05606d5be9ca/93b6b82e-baf8-4adc-8998-0df20831be6d" rel="noopener" target="_blank">our data deletion request form</a>. Once you submit the form, you will receive a verification email with further instructions.</p>
+    <p>Heads up! Deleting your account data will remove any certificates/badges that you’ve earned on Aquent Gymnasium. We advise that you save them now.</p>
```
Comment (jgagne) [inline static-pages/faq/index.html:91]: Consider hyphenation here, as well.

**Review by jgagne — COMMENTED** [review]
Looks good!

**Review by andrewpmiller — APPROVED** [review]
This looks like words! I like them!

**Pull request merged**